Modularize procedures for DNS, DHCP, and TFTP configuration (#2847)
theforeman/foreman-documentation: guides/common/modules/con_configuring-dns-dhcp-and-tftp-additional-resources.adoc

Reviewer: This should probably be ref and not con, even though I guess it is not that important.

[id="configuring-dns-dhcp-and-tftp-additional-resources_{context}"]
= Additional resources

ifndef::foreman-deb,orcharhino[]

> * For information about configuring DNS, DHCP, and TFTP externally, see xref:configuring-external-services[].

Reviewer: optional
```suggestion
* To learn about configuring DNS, DHCP, and TFTP externally, see xref:configuring-external-services[].
```

Author: This would be shorter... but then again, I think throughout the docs set, "For information about" is the standard way of introducing additional resources. So let's stick with this wording.

Reviewer: Usually it is "For more information", "for information" just sounds to me... weird :D but it is non-blocking
